### PR TITLE
Grammar fix on line 148 of Intermediate HTML / Form Validation

### PR DESCRIPTION
**Pull request by eltonbautista**
- [X] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [X] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [X] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [X] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [X] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

Just added "that of which" to make the grammar clearer and easier to understand! 

#### 2. Related Issue

Closes #XXXXX

### CODE REVIEW

File: html_css/intermediate_html/form-validations.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ It is also good practice to use a `placeholder` attribute to show users an examp
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
-The pattern attribute can only be used on `<input>` elements. Some input elements already validate data matches a certain pattern. For example the email input field which will make sure a valid email is entered and the url input element which will check to ensure the url starts with http or https:
+The pattern attribute can only be used on `<input>` elements. Some input elements already validate data that of which matches a certain pattern. For example the email input field which will make sure a valid email is entered and the url input element which will check to ensure the url starts with http or https:
```
Comment (CouchofTomato) [inline html_css/intermediate_html/form-validations.md:148]: Can't we just simplify this to "data that matches a certain pattern"?

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
I agree with @CouchofTomato, I think the "of which" is unnecessary.

**Comment by wise-king-sullyman** [comment]
Additionally just so that you know for the future, please reference the lesson being changed in the title of PRs you make.

I've already edited this PR title so there isn't anything you need to do at this time.

**Comment by eltonbautista** [comment]
Okay I can do that. I hit the commit in wise-king-sullyman's suggestion, does that create the change or do I have to do it from the terminal on my computer? Sorry quite new at this.

**Comment by wise-king-sullyman** [comment]
Hitting the "commit this suggestion" button automatically adds it to the branch PR. No need to be sorry for asking a question 🙂

**Review by wise-king-sullyman — COMMENTED** [review]
Good change, thanks!

**Pull request merged**